substrafl/experiment.py

```diff
@@ -205,6 +214,126 @@ def _get_packages_versions() -> dict:
     }
 
 
+def simulate_experiment(
+    *,
+    client: substra.Client,
+    strategy: ComputePlanBuilder,
+    train_data_nodes: List[TrainDataNodeProtocol],
+    aggregation_node: Optional[AggregationNodeProtocol] = None,
+    evaluation_strategy: Optional[EvaluationStrategy] = None,
+    num_rounds: Optional[int] = None,
+    clean_models: bool = True,
+) -> (SimuPerformancesMemory, SimuStatesMemory, SimuStatesMemory):
```

Comment:
Add all missing parameters from execute experiment, and send a warning if used in simulate experiment